### DISCUSSION TITLE
Support for Airspy HF+

### DISCUSSION THREAD

**Discussion by b3rtdb** [jekhokie/raspberry-noaa-v2#314] [Ideas]
Hi, would it be possible to include the support for the Airspy HF+ SDR receiver?
I tried with GQRX but wasn't able to get it working..

thx and keep up the good work!

**Comment by jekhokie** [comment]
Hi @b3rtdb - unfortunately I don’t have one of those to test with. Have you tried just setting up the framework and using the rtl_fm version of the audio processor/does it not just work out of box?

It might help to come over to Discord and chat with us about what you’ve tried and what did/did not work so we can help troubleshoot - there are a bunch of us there who can try to help: https://discord.gg/368C6PuC

**Comment by dom-robinson** [comment]
@b3rtdb The release to master yesterday defaults the system to using gnuradio based python for both noaa and m2 capture. There is included documentation (in the /scripts/audio_processors/ folder ) that explains how this works. You can now work up your own gnuradio workflows and integrate them with RN2, and this means you can support any hardware that GNURadio can support (although at the moment we are waiting for community to contribute working setups for anything other than RTL_SDR. Hopefully that will come soon!
